app/src/main/java/com/framgia/englishconversation/screen/conversationdetail/ConversationDetailViewModel.java

```diff
@@ -80,10 +83,17 @@ public void onItemUserNameClick(TimelineModel item) {
                     mDetailActivity.startActivity(ProfileUserActivity.getInstance(mDetailActivity,
                             item.getCreatedUser()));
                 }
+
+                @Override
+                public boolean onItemLongClick(View viewGroup, TimelineModel item) {
+                    return false;
+                }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.